db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -13,9 +13,13 @@ def cloud?(prov_type)
 end
 
 def calculate_requested(options_hash = {})
-  {:memory  => get_total_requested(options_hash, :vm_memory) * 1024**2,
+  storage = get_total_requested(options_hash, :storage)
+  if %w(google).include?(@miq_request.source.vendor)
+    storage = get_option_value(@miq_request, :boot_disk_size).gigabytes
+  end
+  {:storage => storage,
+   :memory  => get_total_requested(options_hash, :vm_memory) * 1024**2,
```

Comment:
@billfitzgerald0120 
Should this also be using the gigabytes or megabytes based on the use case instead of doing 1024**2